bin/start-pgbouncer-as-service

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env bash
+# vim: ft=bash
+
+set +e
+
+export PGBOUNCER_PROCESS_REGEX="[v]endor/pgbouncer/bin/pgbouncer"
+export PGBOUNCER_BUILD_PACK_PID
+
+function inf() {
+  echo -e "INFO:  $* "
+}
+
+function err() {
+   echo -e "ERROR: $* "
+}
+
+function is-enabled() {
+  ( shopt -s extglob nocasematch
+    [[ $1 == @(1|true|yes|on) ]]
+  )
+}
+
+function pgbouncer-current-pid() {
+  /bin/ps -eopid,args | grep -E "${PGBOUNCER_PROCESS_REGEX}" | awk '{print $1}'
+}
+
+function is-pgbouncer-running() {
+  local current_pid
+  current_pid=$(pgbouncer-current-pid)
+
+  [[ -n ${current_pid} && ${current_pid} -gt 0 ]] && {
+    inf "pgbouncer-is-running: pid=${current_pid}"
+    return 0
+  }
+
+  inf "pgbouncer-is-not-running"
+  return 1
+}
+
+function is-pgbouncer-service-enabled() {
+  if ! is-enabled "${PGBOUNCER_ENABLED}"; then
+    inf pgbouncer-is-disabled
+    return 1
+  fi
+  return 0
+}
+
+function main() {
+  is-pgbouncer-service-enabled || return 1
```

Comment:
This logic feels unintuitive but I can't really figure out what we are guarding against. I would add a comment here describing why we are early returning. Or, improve the variable names and/or logical operators to make it more clear what is going on.